teamcity_build_scripts/18-mortgage.sh

```diff
@@ -0,0 +1,10 @@
+python3 run_ibis_tests.py --env_name ${ENV_NAME} --env_check True --save_env True --python_version 3.7 -task benchmark        \
+                          --ci_requirements "${PWD}"/ci_requirements.yml                                                      \
+                          --ibis_path "${PWD}"/../ibis/                                                                       \
+                          -executable "${PWD}"/../omniscidb/build/bin/omnisci_server                                          \
```

Comment:
```suggestion
                          --ibis_path "${PWD}/../ibis/"                                                                       \
                          -executable "${PWD}/../omniscidb/build/bin/omnisci_server"                                          \
```
I know it works the way you did, but IMHO it looks cleaner when the whole path is quoted

Reply:
not a problem. Will do it.